src/HaLayout.js

```diff
@@ -7,7 +7,7 @@ import { MyBreadcrumb } from './MyBreadcrumb'
 
 const HaLayout = props => (
   <AppLocationContext>
-    <Layout {...props} appBar={HaAppBar} menu={HaMenu} breadcrumb={MyBreadcrumb} />
+    <Layout {...props} appBar={() => <></>} menu={HaMenu} breadcrumb={MyBreadcrumb} />
```

Comment:
C2. use null instead of ReactElement that returns empty Fragment
 
is : () => <></> really necessary ?
maybe you  can put `null` instead